### ISSUE TITLE
Log config is null

### ISSUE THREAD

**Issue by theit8514** [bcicen/docker-replay#6]
On my docker instance, most of my containers have this in their inspect:
```
            "LogConfig": {
                "Type": "db",
                "Config": null
            }
```

When running these containers on docker-replay, I get an AttributeError:
```
Traceback (most recent call last):
  File "/opt/bin/docker-replay", line 11, in <module>
    sys.exit(main())
  File "/opt/lib/python2.7/site-packages/docker_replay/__init__.py", line 49, in main
    print(DockerReplay(args.container, args.pretty_print))
  File "/opt/lib/python2.7/site-packages/docker_replay/__init__.py", line 21, in __init__
    self.parser = OptionParser(inspect)
  File "/opt/lib/python2.7/site-packages/docker_replay/opts.py", line 14, in __init__
    self.read_opts()
  File "/opt/lib/python2.7/site-packages/docker_replay/opts.py", line 23, in read_opts
    self.opts += list(build_opt(o_type, o_name, self.get(o_key)))
  File "/opt/lib/python2.7/site-packages/docker_replay/models.py", line 9, in build_opt
    for k,v in o_val.items():
AttributeError: 'NoneType' object has no attribute 'items'
```

**Comment by theit8514** [comment]
This is the only log driver provided on my Synology DSM 6.1.5 version of docker, and it's not part of the built-in drivers list. I assume it's a custom driver for DSM so that the web interface can get the logs.

```
# docker info | grep 'Logging Driver'
Logging Driver: db
```

**Comment by bcicen** [comment]
Thanks; this error should now be resolved in the [latest 1.4 release](https://pypi.python.org/pypi/docker-replay/1.4)
